faq.html

```diff
@@ -0,0 +1,39 @@
+---
+permalink: /faq.html
+layout: default
+title: FAQ
+description: Frequently Asked Questions
+---
+
+<div class="faq">
+  <section class="faq__main-content">
+    <div class="wrapper">
+      <h2 class="title-2">{{site.data.faq.title}}</h2>
+      <p class="faq__description">{{site.data.faq.description}}</p>
+      <div class="faq__table">
+        {% for detail in site.data.faq.details %}
+          <details class="">
```

Comment:
Please, make a smooth animation for the details element. 
here is an example:
https://codepen.io/jgustavoas/pen/rNQyxWa?editors=1100

Reply:
Ok, I will add this functionality.